editor/post-format/index.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import { get, find, flowRight } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { withInstanceId, withAPIData } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import { getEditedPostAttribute, getSuggestedPostFormat, getCurrentPostType } from '../selectors';
+import { editPost } from '../actions';
+
+const POST_FORMATS = [
+	{ id: 'aside', caption: __( 'Aside' ) },
+	{ id: 'gallery', caption: __( 'Gallery' ) },
+	{ id: 'link', caption: __( 'Link' ) },
+	{ id: 'image', caption: __( 'Image' ) },
+	{ id: 'quote', caption: __( 'Quote' ) },
+	{ id: 'standard', caption: __( 'Standard' ) },
+	{ id: 'status', caption: __( 'Status' ) },
+	{ id: 'video', caption: __( 'Video' ) },
+	{ id: 'audio', caption: __( 'Audio' ) },
+	{ id: 'chat', caption: __( 'Chat' ) },
+];
+
+function PostFormat( { postType, onUpdatePostFormat, postFormat = 'standard', suggestedFormat, instanceId } ) {
+	if ( ! get( postType.data, [ 'supports', 'post-formats' ] ) ) {
+		return null;
+	}
+
+	const postFormatSelectorId = 'post-format-selector-' + instanceId;
+	const suggestion = find( POST_FORMATS, ( format ) => format.id === suggestedFormat );
```

Comment:
Tip: `lodash#find` allows `find( xs, { id: suggestedFormat } )` syntax — unless you want to keep as-in to keep it closer to the `Array#find` standard.

Reply:
Also, not sure where, but I think I've read that this syntax might be deprecated in lodash 5